src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/InnvilgeVedtak.tsx

```diff
@@ -113,7 +104,7 @@ export const InnvilgeVedtak: React.FC<{
                 ? lagretInnvilgetVedtak?.inntekter
                 : [tomInntektsperiodeRad()],
             samordningsfradragType: lagretInnvilgetVedtak?.samordningsfradragType || '',
-            yngsteBarnFødselsdato: sjekkToggleUtledYngsteBarnetsFødselsdato(vilkår) || '',
+            yngsteBarnFødselsdato: utledYngsteBarnFødselsdato(vilkår) || '',
```

Comment:
```suggestion
            yngsteBarnFødselsdato: utledYngsteBarnFødselsdato(vilkår),
```
Trenger nok ikke `|| ' '` nå lenger